src/Microsoft.OData.Extensions.Migration/Microsoft.OData.Extensions.Migration.Release.nuspec

```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>Microsoft.OData.Extensions.Migration</id>
     <title>Microsoft Extensions OData Migration</title>
-    <version>$VersionNuGetSemantic$</version>
+    <version>1.0.0</version>
```

Comment:
>1.0.0 [](start = 13, length = 5)

however, i don't think it's the right place to hard code it. 

And, it maybe only affect the nuget package version, not the assembly version. I mean the assembly version is still beta

Reply:
Introduced VersionNugetSemanticNoRelease for the nuget versions without the release number